debian/changelog

```diff
@@ -1,3 +1,10 @@
+docker.io (20.10.21-0ubuntu4) lunar; urgency=medium
```

Comment:
Leave the target release as `UNRELEASED` since we are not going to upload this single change.

Reply:
Thanks for your advice. Done